packages/aws-cdk-lib/cx-api/lib/features.ts

```diff
@@ -993,6 +994,19 @@ export const FLAGS: Record<string, FlagInfo> = {
     introducedIn: { v2: '2.124.0' },
     recommendedValue: true,
   },
+
+  //////////////////////////////////////////////////////////////////////
+  [CODEPIPELINE_CROSS_ACCOUNT_KEYS_DEFAULT_VALUE_TO_FALSE]: {
+    type: FlagType.ApiDefault,
+    summary: 'Enables Pipeline to set the default value for crossAccountKeys to false.',
+    detailsMd: `
+      If this is set, and a \`crossAccountKeys\` prop in a \`Pipeline\` construct is not passed to,
+      the construct will set the default value of the prop to false.
+    `,
+    introducedIn: { v2: 'V2NEXT' },
```

Comment:
```suggestion
    introducedIn: { v2: '2.124.0' },
```

Reply:
I think `V2NEXT` is the correct value here according to this: https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md#feature-flags:~:text=Set%20introducedIn.v2%20to%20the%20literal%20string%20%27V2NEXT%27

Reply:
Interesting, I wonder why all the existing definition in this file use specific version.

Reply:
Revoking this comment since Paul was right there, we should just use `V2NEXT`.